neuralforecast/losses/pytorch.py

```diff
@@ -798,6 +798,28 @@ def get_distribution(
                 distr, [AffineTransform(loc=loc, scale=scale)]
             )
 
+    def get_params(
```

Comment:
Can we add an issue with the pending work of homogenization of the `get_params` and the `get_distribution` methods?